{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/emails/templates/emails/password-reset.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'emails/base.html' %}
```

Comment:
Django convention is to use underscores to separate words in template names. This pull request uses dashes inconsistently, too.

Reply:
^^ I think you may have pushed up the fix for this one about five seconds before I wrote this comment :)